lib/winston/transports/console.js

```diff
@@ -49,6 +50,8 @@ module.exports = class Console extends TransportStream {
       if (console._stderr) {
         // Node.js maps `process.stderr` to `console._stderr`.
         console._stderr.write(`${info[MESSAGE]}${this.eol}`);
+        if (inspector.console.hasOwnProperty(info[LEVEL]))
+          inspector.console[info[LEVEL]](info[MESSAGE]);
```

Comment:
Log levels like `http` or `emerg` or any custom log levels will go nowhere, I think?  Would maybe want `(inspector.console[info[LEVEL]] || inspector.console.log)(info[MESSAGE])`?  Except even that won't work, as if anyone has a custom log level named `trace` or `profile`, it's going to be calling a function which does something different (accumulate memory leaks and crash =).  Probably would want an explicit `inspector.console.error` on the `stderr` case and `inspector.console.log` in the `stdout` case?

Reply:
> Probably would want an explicit inspector.console.error on the stderr case and inspector.console.log in the stdout case?

This is definitely not desirable since levels are colored and displayed differently. I've updated the PR with a new option to provide this mapping as well as a default for it that covers most cases.